docs/opcua-configuration.md

```diff
@@ -128,8 +128,8 @@ generator tool](https://github.com/OPCFoundation/Misc-Tools), or Akri's [certifi
 
 Finally, when mounting certificates is enabled with with Helm via `--set opcua.mountCertificates='true'`, the
 secret named `opcua-broker-credentials` will be mounted into the OPC UA brokers. It is mounted to the volume
-`credentials` at the `mountPath` /etc/opcua-certs/client-pki, as shown in the [OPC UA monitoring Helm
-template](../deployment/helm/templates/opcua-monitoring.yaml). This is the path where the broker expects to find the
+`credentials` at the `mountPath` /etc/opcua-certs/client-pki, as shown in the [OPC UA Helm
+template](../deployment/helm/templates/opcua.yaml). This is the path where the broker expects to find the
```

Comment:
should i see this helm template file change in this PR?

Reply:
nevermind, it looks like the file was already renamed ... this is just matching the docs to that.

Reply:
Yeah this PR is trying to fix that oops